servlet/5.0/_index.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: "Jakarta Servlet 5.0 (milestone)"
-date: 2020-06-12
+title: "Jakarta Servlet 5.0"
+date: 2020-07-20
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Servlet defines a server-side API for handling HTTP requests and responses.
 
 * [Jakarta Servlet 5.0 Release Record](https://projects.eclipse.org/projects/ee4j.servlet/releases/5.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Servlet 5.0 Specification Document](./servlet-spec-5.0-SNAPSHOT.pdf) (PDF)
-* [Jakarta Servlet 5.0 Specification Document](./servlet-spec-5.0-SNAPSHOT.html) (HTML)
+* [Jakarta Servlet 5.0 Specification Document](./servlet-spec-5.0.pdf) (PDF)
```

Comment:
This should have jakarta- prefix

Please see PR https://github.com/eclipse-ee4j/servlet-api/pull/353

Reply:
Jakarta prefix added. Can you check again?